omoefe-dukuye/SendIT
# #162036914 Add route for  destination change

#### What this PR does
If merged, this pull request will
- Create PUT /api/v1/parcels/:parcelId/destination endpoint
- Add validation for it

#### How to test manually
- Clone the repo `git clone https://github.com/un-tethered/SendIT.git`
- Check out the branch `ft-user-change-destination-162036914`
- Install Postgresql
- Create a database named `SendIT`
- Run `npm install` to install application dependencies
- Run `babel-node db.js createAllTables` to create tables
- Start server with `npm run dev` 
- Test endpoints with Postman

#### Pivotal tracker story
[#162036914](https://www.pivotaltracker.com/story/show/162036914)
